src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [inputData, setFilteredList] = useState('');
+  const filteredList = moviesFromServer.filter(
```

Comment:
The filtered list of movies should be stored in a variable named visibleMovies instead of filteredList, as per the task description.
![image](https://github.com/mate-academy/react_movies-list-filter/assets/99403198/558c9ff5-b97a-426e-912d-0a4c41094c80)